Implemented Kover Features artifact to integration with Android Gradle Plugin (#536)
Kotlin/kotlinx-kover: docs/offline-instrumentation/index.md

### Class instrumentation

#### Instrumentation by Kover CLI
The Kover CLI is a fat jar that needs to be called and passed certain commands through arguments.

For instrumentation, you must first build the application, then the root directories for the class files
must be passed to Kover CLI as arguments, see [Kover CLI](../cli#offline-instrumentation) for the technical details.

#### Instrumentation by Kover Features

> Kover Features is an artifact with Java classes that provide capabilities of the Kover library.

Reviewer: ```suggestion
Kover Features is a library that provides capabilities similar to Kover CLI and Kover Gradle plugin.
```

> Dependency on Kover Features `org.jetbrains.kotlinx:kover-features-java:0.7.5`.

Reviewer: ```suggestion
You can declare a dependency on Kover Features using following coordinates: `org.jetbrains.kotlinx:kover-features-java:0.7.5`.
```

> Then you need to use the Kover Features classes to instrument the bytecode of each class:

Reviewer: ```suggestion
Then you can use the Kover Features classes to instrument the bytecode of each class:
```

```kotlin
import kotlinx.kover.features.java.KoverFeatures
  // ...

  val instrumenter = KoverFeatures.createOfflineInstrumenter()
  
  // read class-file with name `fileName` bytes to `classBytes`
  val instrumentedBytes = instrumenter.instrument(classBytes, fileName)
  // save `instrumentedBytes` to file
```

### Dump coverage result

> To run classes instrumented offline, you'll need to add `org.jetbrains.kotlinx:kover-offline-runtime` artifact to the application's classpath.

Reviewer: ```suggestion
To run classes instrumented offline (with CLI) or programmatically (with Kover Features), you'll need to add `org.jetbrains.kotlinx:kover-offline-runtime` artifact to the application's classpath.
```
